transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/UpdateTransferMethodFragment.java

```diff
@@ -413,17 +414,32 @@ public void showTransactionInformation(@NonNull List<Fee> fees,
         if (isFeeAvailable(fees) && isProcessingTimeAvailable(processingTime)) {
             String formattedFee = FeeFormatter.getFormattedFee(header.getContext(), fees);
             feeAndProcessingTime.setVisibility(View.VISIBLE);
-            feeAndProcessingTime.setText(
-                    feeAndProcessingTime.getContext().getString(R.string.feeAndProcessingTimeInformation, formattedFee,
-                            processingTime.getValue()));
+            if (isZeroFeeAvailable(feeAndProcessingTime.getContext(), formattedFee)) {
```

Comment:
too many if/else conditions in this method too, please refactor here too

Reply:
move to common method in FeeFormatter as getFormattedFeeAndProcessingTime()